src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JdepsMerger.kt

```diff
@@ -90,9 +90,23 @@ class JdepsMerger {
       }
 
       if (reportUnusedDeps != "off") {
-        val unusedLabels = dependencyMap.values
-          .filter { it.kind == Deps.Dependency.Kind.UNUSED }
-          .mapNotNull { readJarOwnerFromManifest(Paths.get(it.path)).label }
+        val kindMap = mutableMapOf<String, Deps.Dependency.Kind>()
```

Comment:
@mauriciogg is there any test coverage for this behavior in `JdepsMergerTest` that can be added?